src/components/Main/Main.jsx

```diff
@@ -0,0 +1,252 @@
+                    return +a["Years of experience"].split(',').join('') > +b["Years of experience"].split(',').join('') ? 1 : -1
+                }
+            });
+        }
+        else if (arg == "expDown") {
+            rows.sort((a, b) => {
+                if (a["Years of experience"] !== undefined && b["Years of experience"] !== undefined) {
+                    return +b["Years of experience"].split(',').join('') > +a["Years of experience"].split(',').join('') ? 1 : -1
+                }
+            });
+        }
+
+        setRows((prev) => [...prev, rows]);
+    }
+
+    const searching = (value) => {
+        setRows(copyOfRows);
+
+        if (value !== '') {
+            setRows((prev) => prev.filter((item) => {
+                return (
+                    item['Date'].includes(value.toLowerCase()) || item['Company'].toLowerCase().includes(value.toLowerCase()) ||
+                    item['Position'].toLowerCase().includes(value.toLowerCase()) || item['Salary'].includes(value.toLowerCase()) ||
+                    item['Years of experience'].includes(value.toLowerCase())
+                )
+            }));
+        }
+    }
+
+    return (
+        <main>
+
+            <About />
+            <div className="container">
+                <div className="form-btn-wrapper mt30">
+                    <a href="https://docs.google.com/forms/d/1M4ztN09EvaminyLIDH4rOgtnr0lW-AHEYXiThbpAZa0/viewform?edit_requested=true" className="btn dark">Submit my report</a>
+                    <form action="">
+                        <label htmlFor="search" hidden>Search</label>
+                        <div className="input-wrapper">
+                            <img className="search-icon" src={searchIcon} alt="Search" />
+                            <input
+                                className="search-input"
+                                type="text"
+                                placeholder="Search"
+                                onChange={(e) => {
+                                    searching(e.target.value);
+                                }}
+                            />
+                            <p>Showing {rows.length} of {copyOfRows.length != 0 ? copyOfRows.length : rows.length} entries</p>
+                        </div>
+                    </form>
+                </div>
+            </div>
+
+            {
+                rows.length > 0 ?
+                    <main>
+                        <div className="container">
+                            <div className="table">
+                                <div className="row table-header">
+
+                                    <div className="col col-label">
+                                        <button
+                                            className={sortType == "dateUp" ? "sort-btn up" : sortType == "dateDown" ? "sort-btn down" : "sort-btn"}
+                                            onClick={() => {
```

Comment:
This can simplified and use strict comparing "===" instead of "=="

const handleBtnClick = () => {
                                                setSortType(sortType === "dateUp" ? "dateDown" : "dateUp");
                                                sortTable(sortType === "dateUp" ? "dateDown" : "dateUp");
                                            }

onClick={handleBtnClick}